1) If you select one jelly bean at random from a bag, what is the probability that the bean is not Licorice?


In [5]:
49/50

0.98

2) What is the probability that a randomly-selected bean is Top Banana, Toasted Marshmallow, or Coconut?

In [2]:
3/50

0.06

3) True or false: if you roll two six-sided dice, the probability that at least one die will show a six is one third. Why or why not?


In [22]:
False

P(A) = 1/6 

False

4) Suppose you shuffle two decks of cards together and draw a single card at random from the deck. What is the probability that the card will be both a club and an eight, nine, or ten?


In [6]:
(1/52) + (1/52) + 1/(52)

0.057692307692307696

5) A recent survey of employees at a firm showed that 55% drink coffee, 25% drink tea, and 35% drink neither. How many of those surveyed drink both coffee and tea?

Remember the following two equations for the next two problems

$P(A \bigcup B) = P(A) + P(B) - P(A)P(B)$

$P(A^c) = 1 - P(A)$

In [17]:
pa = 55
pb = 25
not_pa_and_pb = 35
pa_and_pb = 100 - not_pa_and_pb

pa_or_pb = pa + pb -pa_and_pb
pa_or_pb

15

6) A university survey on mobile computing finds that 85% of their students own smartphones, 30% own tablets, and 25% own both. What percentage of students own neither?

In [19]:
pa = 85
pb = 30
pa_and_pb = 25
pa_or_pb = pa + pb - pa_and_pb
not_pa_or_pb = 100 - pa_or_pb
not_pa_or_pb

10

7) Among the guests who stay at a casino hotel, 30% play poker, 40% play blackjack, 10% play both poker and blackjack, 15% play craps, and 25% do not gamble at all. True or false: playing poker and playing craps are mutually exclusive.
